test/view/drop_while_view.cpp

```diff
@@ -0,0 +1,71 @@
+// cmcstl2 - A concept-enabled C++ standard library
+//
+//  Copyright Casey Carter
+//  Copyright Eric Niebler
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/caseycarter/cmcstl2
+//
+#include <stl2/view/drop_while.hpp>
+#include <stl2/view/iota.hpp>
+#include <stl2/view/take.hpp>
+#include <list>
+#include "../simple_test.hpp"
+
+namespace ranges = __stl2;
```

Comment:
Something cmcstl2-specific? (Similarly in other places).

Reply:
The tests are copied from CMCSTL2, and most of them are still unused -- as I implement them I activate the tests and hack it enough to make it compile (and hopefully pass). (You can see that most of them are still disabled in CMakeLists.txt.)